external/docker/spark-test/base/Dockerfile

```diff
@@ -15,14 +15,14 @@
 # limitations under the License.
 #
 
-FROM ubuntu:precise
+FROM ubuntu:xenial
```

Comment:
This is probably a good change, but is it required in order to get Java 8?

Reply:
Yes. It's required. When we change only jdk7 to jdk8, it fails.

Reply:
yea no 8 on precise